modules/administration-guide/pages/configuring-single-and-multiroot-workspaces.adoc

```diff
@@ -0,0 +1,91 @@
+:_content-type: PROCEDURE
+:description: Configuring single and multiroot workspaces
+:keywords: singleroot, multiroot, workspace
+:navtitle: Configuring single and multiroot workspaces
+// :page-aliases:
+
+[id="configuring-single-and-multiroot-workspaces"]
+= Configuring single and multi root workspaces
+
+With the multi-root workspace feature, you can work with multiple project folders in the same workspace. This is useful when you are working on several related projects at once, such as product documentation and product code repositories.
+
+TIP: See link:https://code.visualstudio.com/docs/editor/workspaces[What is a VS Code "workspace"] for better understanding and authoring the workspace files.
+
+[NOTE]
+====
+Starting with {prod-short} 7.73.0, the workspace is set to open in multi-root mode by default.
+====
+
+Once workspace is started, the `/projects/.code-workspace` workspace file is generated. The workspace file will contain all the projects described in the devfile.
+
+[source,json]
+----
+{
+	"folders": [
+		{
+			"name": "project-1",
+			"path": "/projects/project-1"
+		},
+		{
+			"name": "project-2",
+			"path": "/projects/project-2"
+		}
+	]
+}
+----
+
+If the workspace file already exist, it will be updated and all missing projects will be taken from the devfile.
+If you remove a project from the devfile, it will be left in the workspace file.
+
+You can change the default behavior and provide your own workspace file or switch to a single-root workspace.
+
+.Procedure
+
+* Provide your own workspace file.
+
+** Put a workspace file with the name `.code-workspace` into the root of your repository. After workspace creation, the Visual Studio Code - Open Source ("Code - OSS") will use the workspace file as it is.
++
+[source,json]
+----
+{
+	"folders": [
+		{
+			"name": "project-name",
+			"path": "."
+		}
+	]
+}
+----
++
+[IMPORTANT]
+====
+Be careful when creating a workspace file. In case of errors, an empty Visual Studio Code - Open Source ("Code - OSS") will open instead.
```

Comment:
```suggestion
Be careful when creating a workspace file. In case of errors, an empty Visual Studio Code - Open Source ("Code - OSS") will open instead.
```
```suggestion
Be careful when creating a workspace file. In case of errors, an empty Visual Studio Code - Open Source ("Code - OSS") will be opened instead.
```